ttg/ttg/parsec/ttg.h

```diff
@@ -1429,6 +1511,11 @@ namespace ttg_parsec {
           parsec_list_item_ring_push_sorted(&(*task_ring)->super, &task->parsec_task.super,
                                             offsetof(parsec_task_t, priority));
         }
+      } else if constexpr (!ttg::meta::is_void_v<keyT>) {
```

Comment:
The PaRSEC backend doesn't support void keys. Is that general policy or do we need to fix that? Do we make sure in the upper layers that pull terminals are not used with void keys?

Reply:
Added a static_assert in Edge constructor to take care of this.